pandoc-cli/man/pandoc.1

```diff
@@ -3140,7 +3140,7 @@ mainfontoptions:
 fonts to try if a glyph isn\(cqt found in \f[CR]mainfont\f[R],
 \f[CR]sansfont\f[R], or \f[CR]monofont\f[R] respectively.
```

Comment:
We regenerate man pages just once before release, so don't include this in the Pr.